src/app/docs/text-expand/text-expand-docs.component.html

```diff
@@ -0,0 +1,32 @@
+<sky-docs-demo-page
+  moduleName="SkyTextExpandModule"
+  moduleSourceCodePath="src/app/public/modules/text-expand/"
+  pageTitle="Text expand"
+>
+  <sky-docs-demo-page-summary>
+    The text expand component truncates long blocks of text with an ellipsis and a link to expand the text.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-demo>
+    <sky-text-expand
+      [text]="longText"
+    >
+    </sky-text-expand>
+  </sky-docs-demo>
+
+  <sky-docs-code-examples>
+    <sky-docs-code-example
+      heading="Text expand inline"
```

Comment:
Do we really need `Text expand` in these headings? I think it might be clearer and easier to read at a glance if we went with just `Inline`, `Modal view`, and `Modal view with newline characters`.

Reply:
Updated.